tox.ini

```diff
@@ -4,7 +4,7 @@ skipsdist = True
 
 [testenv]
 usedevelop = True
-commands = py.test {posargs:tests/} --assert=plain
```

Comment:
Right, but it's a bit hard to do that if you want to run the same test on different Python versions :)